ajaxhookPosition.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+* 2007-2014 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Academic Free License (AFL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/afl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2014 PrestaShop SA
+* @license   http://opensource.org/licenses/afl-3.0.php Academic Free License (AFL 3.0)
+* International Registered Trademark & Property of PrestaShop SA
+*/
+
+include(dirname(__FILE__).'/../../config/config.inc.php');
+include_once(_PS_CLASS_DIR_.'/../classes/Customer.php');
+include(dirname(__FILE__).'/sendinblue.php');
+
+if (Tools::getValue('token') != Tools::encrypt(Configuration::get('PS_SHOP_NAME')))
+	die('Error: Invalid Token');
+
+$sendin = new Sendinblue();
+$sendin->hooksPosition(Tools::getValue('blockname'), Tools::getValue('blockvalue'), Tools::getValue('id_shop_group'), Tools::getValue('id_shop'));

```

Comment:
All these values have never been tested. What if they are not defined when the script is called ?